cisco_sdwan/metadata.csv

```diff
@@ -0,0 +1,32 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+cisco_sdwan.bfd_session.status,gauge,,,,The status of the BFD session.,0,cisco_sdwan,,
+cisco_sdwan.control_connection.status,gauge,,,,The status of the control connection.,0,cisco_sdwan,,
+cisco_sdwan.cpu.usage,gauge,,percent,,Percentage of CPU currently being used.,0,cisco_sdwan,,
+cisco_sdwan.crash.count,count,,,,The number of crashes.,0,cisco_sdwan,,
+cisco_sdwan.device.reachable,gauge,,,,The status of the device.,0,cisco_sdwan,,
+cisco_sdwan.device.uptime,gauge,,,,The time (in hundredths of a second) since the network management portion of the system was last re-initialized.,0,cisco_sdwan,,
+cisco_sdwan.disk.usage,gauge,,percent,,The percentage of disk currently being used.,0,cisco_sdwan,,
+cisco_sdwan.interface.rx_bandwidth_usage,gauge,,percent,,The percent rate of used received bandwidth.,0,cisco_sdwan,,
+cisco_sdwan.interface.rx_bits,count,,bit,,The total number of bits received on the interface.,0,cisco_sdwan,,
+cisco_sdwan.interface.rx_bps,gauge,,bit,second,The inbound bandwidth rate in kilobit per second.,0,cisco_sdwan,,
+cisco_sdwan.interface.rx_drops,count,,packet,,The number of inbound packets chosen to be discarded even though no errors had been detected to prevent them being deliverable to a higher-layer protocol.,0,cisco_sdwan,,
+cisco_sdwan.interface.rx_errors,count,,packet,,The number of inbound packets that contained errors preventing them from being deliverable to a higher-layer protocol.,0,cisco_sdwan,,
+cisco_sdwan.interface.speed,gauge,,,,The interface current speed in bits per second.,0,cisco_sdwan,,
```

Comment:
should this one be bit per second as well?

Reply:
It could, but setting it as bit per second makes the unit scale to `Ki/Mi/Gi` units in the UI which is very weird for network interfaces (1Gb interface will be displayed as 0,9313 GiB), so we choose to not set units (similarly to [SNMP](https://github.com/DataDog/integrations-core/blob/656df2b5280f25964a34b903bee58d2c22d9d4c8/snmp/metadata.csv#L1019))